security-proxy/src/deb/resources/etc/georchestra/security-proxy/user-agents.properties

```diff
@@ -0,0 +1,5 @@
+useragent0.value=.*ArcGIS.*
+useragent1.value=.*uDig.*
+useragent2.value=.*QGIS.*
```

Comment:
The above 3 lines should be removed (or at least commented out)
See https://github.com/georchestra/georchestra/blob/15.06/security-proxy/src/main/webapp/WEB-INF/applicationContext-security.xml#L52-L56